app/providers/services/message/index.js

```diff
@@ -12,12 +14,14 @@ class MessageService {
 
     messageParams.t = this.helpers.currentTimeStamp()
 
-    const message = await this.messageRepo.create(messageParams)
+    if (messageParams.encrypted_message_type !== undefined) {
+      messageParams.expired_at = new Date()
```

Comment:
let's get rid of this field
instead - create TTL index by `created_at` and use `partialFilterExpression ` to filter encrypted messages only